src/scripts/record.js

```diff
@@ -91,6 +102,23 @@ const ensureDirectoryExists = (directory) => {
   }
 };
 
+const insertFrameStatsHeaderIndexes = (line) => {
+  if (Object.keys(framestatsHeaderIndexes).length > 0) return;
+  const deviceFrameStatsHeader = line
+    .replaceAll(' ', '')
+    .split(',')
+    .filter((str) => str !== '');
```

Comment:
```suggestion
  const deviceFrameStatsHeader = line
    .replaceAll(' ', '')
    .split(',')
    .filter((str) => str !== '');
```

If this is a very frequent operation then we should consider performing these steps in single loop.

Reply:
This is a one time activity